packages/jest-runtime/src/__tests__/instrumentation.test.js

```diff
@@ -12,6 +12,7 @@ import vm from 'vm';
 import path from 'path';
 import os from 'os';
 import ScriptTransformer from '../ScriptTransformer';
+import {normalize} from 'jest-config';
```

Comment:
I recommend to also import `defaults` and spread over it.

```js
  const config = normalize(
    {
      ...defaults,
      cache: false,
      cacheDirectory: os.tmpdir(),
      rootDir: '/',
    },
    {},
  ).options;

Reply:
@SimenB done